zad4/app/services/football_player_manager.rb

```diff
@@ -8,14 +8,16 @@ def initialize(player)
   end
 
   def stronger_players
-    # TODO: Implement me pls
+    Player.where("power > ?", @player.power)
```

Comment:
Tu mini uwaga, w tej klasie wcześniej zzdefiniowany był `attr_reader`, którego nie użyłaś ;) Ale nie jest to żaden poważny błąd